.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+version: 2
+updates:
+  - directory: /
+    package-ecosystem: github-actions
+    rebase-strategy: auto
+    schedule:
+      interval: daily
+  - directory: /
+    package-ecosystem: npm
+    rebase-strategy: auto
+    schedule:
+      interval: daily
```

Comment:
I got some feedback from folks on another repo in here that we have been using renovate because dependabot is such trash. I would rather add something than nothing but also just wanted to call it out since this was new for this repo.

Reply:
Fine with either tbh. We can remove this once we get the renovate setup in the repo

Reply:
Maybe @ljharb or @dominykas want to chime in? I feel like it was their feedback but my memory might be off.